src/index.js

```diff
@@ -8,10 +8,7 @@ import * as serviceWorker from "./serviceWorker";
 import reducers from "./reducers";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
-const store = createStore(
```

Comment:
But if you remove these lines we are not able to use Redux dev tools, my point was to only make then available for DEV and not for production